client/js/table-view.js

```diff
@@ -23,26 +26,43 @@ class TableView {
 	}
 
 	normalizeValueForRendering(value) {
-		return value || ''; //returns empty string if argument is undefined
+		return value || '';
 	}
 
 	renderFormulaBar() {
 		const currentCellValue = this.model.getValue(this.currentCellLocation);
 		this.formulaBarEl.value = this.normalizeValueForRendering(currentCellValue);
-		this.formulaBarEl.focus(); //defaults selected element
+		this.formulaBarEl.focus();
 	}
 
 	renderTable() {
 		this.renderTableHeader();
 		this.renderTableBody();
+		this.renderTableFooter();
 	}
+
 	renderTableHeader() {
 		removeChildren(this.headerRowEl);
 		getLetterRange('A', this.model.numCols).map(colLabel => createTH(colLabel)).forEach(th => this.headerRowEl.appendChild(th));
 	}
 
-	isCurrentCell(col, row) {
-		return this.currentCellLocation.col === col && this.currentCellLocation.row === row;
+	renderTableFooter() {
+		removeChildren(this.footerRowEl);
+		for (let col = 0; col < this.model.numCols; col++) {
+			let colSum = '';
```

Comment:
Why should a sum start out being a string?  

I think it's complicating your later processing.

This would be cleaner if you did the processing in stages.

1) gather all the values from the target column
2) filter to just numeric values
3) if you have any, sum them
4) create the element and set to sum, if you have one